hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -2938,7 +2937,7 @@
         clk_i:
         {
           clock: io_div4
-          group: secure
+          group: powerup
```

Comment:
@matutem ah, this will address the clk_i and clk_por_i discrepancy we were talking about earlier today.